__tests__/atlas.test.ts

```diff
@@ -482,7 +482,7 @@ describe('all reports with pull request', () => {
     expect(spyOnError).toHaveBeenCalledTimes(0)
     expect(spyOnNotice).toHaveBeenNthCalledWith(
       1,
-      'Adding a unique index "uniq_name" on table "users" might fail in case column "name" contains duplicate entries',
+      'Adding a unique index "uniq_name" on table "users" might fail in case column "name" contains duplicate entries (MF101)\n\nDetails: https://atlasgo.io/lint/analyzers#MF101',
```

Comment:
Can we replace the “details” with making the “code” a link?

Reply:
Unfortunately, no

![Screen Shot 2022-09-04 at 9 34 42](https://user-images.githubusercontent.com/1522681/188300776-ea643273-933f-4324-b7d9-3bafa8949cfd.png)